server/fishtest/rundb.py

```diff
@@ -548,15 +550,18 @@ def get_finished_runs(
             q["is_green"] = True
         if yellow_only:
             q["is_yellow"] = True
+        if not query_failed:
+            q["failed"] = False
 
-        c = self.runs.find(
-            q, skip=skip, limit=limit, sort=[("last_updated", DESCENDING)]
+        runs_list = self.runs.find(
+            q, skip=skip, limit=limit, sort=[("last_updated", DESCENDING)],
+            projection=projection
         )
 
-        count = self.runs.count_documents(q)
+        q_count = dict(q)
```

Comment:
We want to get the count without the deleted runs, we were not able to get it in a fast way so we took the total count.

Reply:
Okay makes sense, I was wondering why this count is with deleted which doesn't make any sense